ldb/static/partials/ldb/person.html

```diff
@@ -275,9 +275,9 @@
         <a href class="icon-refresh" ng-hide="editmode" ng-click="person.student_model.confirm()"></a>
       </div>
 
-      <div ng-show="person.student_model.graduated && !editmode"><strong>Afgestudeerd</strong></div>
+        <div ng-show="person.student_model.enroled && !editmode"><strong>Ingeschreven</strong></div>
```

Comment:
Ik vind het niet zo handig om dat vinkje bij weergave ineens om te draaien. In de nieuwe template kunnen we het natuurlijk wel duidelijker aangeven.